apps/backend-sls/prisma/seed.ts

```diff
@@ -282,6 +282,50 @@ async function main(prisma: Prisma.PrismaClient) {
     update: {},
   })
 
+  const question6 = await prisma.question.upsert({
+    where: {
+      id: '996f208b-d567-4f1e-8c57-6f555866c33f',
+    },
+    create: {
+      id: '996f208b-d567-4f1e-8c57-6f555866c33f',
+      name: 'RDWachstum',
+      content:
+        'Wie hoch schätzt du das durchschnittliche Wachstum der Forschung- und Entwicklungskosten in den nächsten fünf Jahren?',
+      contentPlain:
+        'Wie hoch schätzt du das durchschnittliche Wachstum der Forschung- und Entwicklungskosten in den nächsten fünf Jahren?',
+      type: 'NUMERICAL',
+      options: {
+        restrictions: {
```

Comment:
the session management branch adds seeding for numerical and free text as well (will conflict). add solutions to the seed

Reply:
naming schemes will be adapted to the session management branch in order to simplify merging